airbyte-integrations/connectors/source-braze/source_braze/manifest.yaml

```diff
@@ -164,6 +164,11 @@ definitions:
       $ref: "#/definitions/datetime_14d_incremental_sync"
     retriever:
       $ref: "#/definitions/non_paginated_retriever"
+      requester:
+        $ref: "#/definitions/requester"
+        request_parameters:
+          include_variant_breakdown: true
```

Comment:
Are additional fields declared in the schema?

Reply:
Yes, additional properties are declared here - https://github.com/airbytehq/airbyte/pull/31607/files#diff-d8a79b4cd6e3c9444fafbbe6071a31d83a364e018bb4782e6e3cf07722862778R4